clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3967,6 +3967,80 @@ Attribute ``trivial_abi`` has no effect in the following cases:
   }];
 }
 
+
+def LifetimeCaptureByDocs : Documentation {
+  let Category = DocCatFunction;
+  let Content = [{
+    Similar to `lifetimebound`_, the ``lifetime_capture_by(X)`` attribute on a function
+parameter or implicit object parameter indicates that that objects that are referred to
+by that parameter may also be referred to by the capturing entity ``X``.
+
+The ``lifetimebound`` attribute on a function parameter or implicit object
```

Comment:
This paragraph is irrelevant.

Reply:
Aah. missed to push this change. Thanks.